packages/framework/src/Model/Product/Search/Export/ProductSearchExportStructureFacade.php

```diff
@@ -29,20 +38,80 @@ public function __construct(Domain $domain, ElasticsearchStructureManager $elast
         $this->elasticsearchStructureManager = $elasticsearchStructureManager;
     }
 
+    /**
+     * @required
+     * @param \Shopsys\FrameworkBundle\Component\Elasticsearch\ElasticsearchStructureUpdateChecker $elasticsearchStructureUpdateChecker
+     * @internal Will be replaced with constructor injection in the next major release
+     */
+    public function setElasticsearchStructureUpdateChecker(ElasticsearchStructureUpdateChecker $elasticsearchStructureUpdateChecker): void
```

Comment:
Please check #1164 in order to make this setter injector consistent with the rest (i.e. making the use of constuctor injection possible).